src/View/SerializedView.php

```diff
@@ -103,10 +103,7 @@ public function render($view = null, $layout = null): ?string
 
             return $result;
         }
-        if ($view !== false && $this->_getViewFileName($view)) {
-            return parent::render($view, false);
-        }
 
-        return null;
+        return parent::render($view, false);
```

Comment:
Can someone confirm this is okay? I don't see any value in the render method returning `null`, for serialized content or otherwise.

Reply:
render method here and otherwise returns 
```php
return $this->Blocks->get('content');
```
and it should returns `''` or a string 
so it is OK imo
just you can have a condition above this line for reducing calling  the parent class 
```php
        if ($view === false ) {
            return '';
        }
```
I dont see any benefit of  `render($view = false):`

Reply:
Meh, let's keep the parent call. Basically if serialization is off let the parent `View` class handle rendering instead of making decisions here.